saleae/saleae.py

```diff
@@ -37,15 +37,44 @@
 PY3K = sys.version_info[0] == 3
 
 @enum.unique
-class Trigger(enum.IntEnum):
-	'''Trigger types to start sampling.'''
+class TriggerType(enum.IntEnum):
```

Comment:
I'm a bit concerned that this renamed will not be backward compatible. For example, if folks had  code like `set_trigger_one_channel(my_channel, saleae.Trigger.High)`, which would be pretty reasonable, it will no longer work.

Reply:
Oh, I've evaluated the compatibility to hastily, you are totally right.

Well, the enum is a bit inconvenient with the new trigger types. But having two distinct classes for same thing is a bit confusing in my opinion. If you prefer, I could rework it to have two different classes as suggested previously.

Reply:
It may be that the right solution here is to just give up making Trigger a subclass of Enum, that's not really providing so much value here. Requires adding a little custom serialization logic to the revised Trigger class, but that seems fine. Then we could just use a single class for all of this functionality.

Reply:
> Requires adding a little custom serialization logic to the revised Trigger class, but that seems fine

Sorry for no response for a long time. I was a bit busy. Can you be a bit more specific about the custom serialization? I'm not familiar with such technique. But if you are able to direct me, I'll try to implement the stuff. :-)

Reply:
I fully understand busy :)

Re-reading my own comment, "custom serialization" is already being done by a function ( [_set_triggers_for_all_channels](https://github.com/ppannuto/python-saleae/blob/master/saleae/saleae.py#L250) ). I imagine I was thinking that It would probably be a cleaner architecture to have the `Trigger` class have a `build` method, or a `to_saleae_string` or some such, but that doesn't need to be part of this PR.